src/els_code_navigation.erl

```diff
@@ -72,38 +71,34 @@ goto_definition(_Filename, _) ->
 find([], _Kind, _Data) ->
   {error, not_found};
 find([Uri|Uris0], Kind, Data) ->
-  case els_db:find(documents, Uri) of
-    {ok, Document} ->
-      POIs = els_document:points_of_interest(Document, [Kind], Data),
-      case POIs of
+  case els_dt_document:lookup(Uri) of
```

Comment:
To support both sets and bags, the lookup API currently returns a list. We may want to change this later.